bodyprocessors/bodyprocessors.go

```diff
@@ -25,6 +25,8 @@ type Options struct {
 	FileMode fs.FileMode
 	// DirMode is the mode of the directory that will be created
 	DirMode fs.FileMode
+	// Strict use strict parsing for XML or others body processors
+	Strict bool
```

Comment:
Not sure about this. Do we really want to increase the API surface for the XML use case?

Reply:
We see many APIs in enterprises still using XML. Seems can't do away with it right now.

Reply:
I guess 2023 is the new 1995